### PR TITLE
Added network state constants

### PR DESCRIPTION
**Pull request by IvanRF**
Documentation:
- https://www.w3.org/TR/html5/embedded-content-0.html#htmlmediaelement
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/networkState

### CODE REVIEW

File: externs/browser/html5.js [hunk 1]
```diff
@@ -1674,6 +1674,30 @@ function HTMLMediaElement() {}
  * @type {number}
  * @const
  */
+HTMLMediaElement.NETWORK_EMPTY;  // = 0
+
+/**
+ * @type {number}
```
Comment (brad4d) [inline externs/browser/html5.js]: Please change these to just `@const {number}`

**Comment by brad4d** [comment]
Just one small change before I import this. See comment on the code.

**Comment by IvanRF** [comment]
@brad4d Done, I updated the other 5 constants definitions too.

**Comment by brad4d** [comment]
I've imported the latest commits & sent them for internal testing and review.

**Comment by brad4d** [comment]
change was submitted internally.
The next push to github should close this PR.

**Pull request closed without merging**